include/modem/lte_lc.h

```diff
@@ -582,6 +582,31 @@ struct lte_lc_conn_eval_params {
 	uint32_t cell_id;
 };
 
+/** @brief Specifies which type of search the modem should perform when a neighbor
+ *	   cell measurement is started.
+ */
+enum lte_lc_neighbor_search_type {
+	/** The modem searches based on previous cell history.
```

Comment:
Maybe it would be good to mention that modem searches for cells only in the network it is registered to. The two other options may also use any other network, if RPLMN cells are not found.
```suggestion
	/** The modem searches RPLMN based on previous cell history.
```